src/Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -4594,23 +4594,26 @@ void RenderOpenGL::DrawBuildingsD3D() {
     if (!decal_builder->bloodsplat_container->uNumBloodsplats) return;
 
     for (BSPModel &model : pOutdoor->pBModels) {
-        bool reachable;
-        if (!IsBModelVisible(&model, 256, &reachable)) {
-            continue;
-        }
-        model.field_40 |= 1;
```

Comment:
Please add a TODO next to `field_40` to remove all usages then.

Reply:
field_40 appears to be a visibility flag. It gets set in a number of places but never appears to be used. Will add a TODO. Might be useful down the line to use to only provide map tool tips of buildings party has been near if we add distance check to when its set.